sqlpad.yaml

```diff
@@ -29,15 +29,19 @@ pipeline:
 
   - working-directory: /home/build/server
     runs: |
+      # Remove mysql2 from the lock to mitigate CVE-2024-21508
+      yarn remove mysql2
+
       # Create "resolutions" section of package.json
       jq '.resolutions |= (if . then . else {} end)' package.json > temp.json && mv temp.json package.json
-      for override in '"semver"="6.3.1"' '"@node-saml/node-saml"="4.0.5"' '"ip"="2.0.1"' '"jose"="4.15.5"' '"xlsx"="https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz"' '"tar"="6.2.1"'; do
+      for override in '"semver"="6.3.1"' '"@node-saml/node-saml"="4.0.5"' '"ip"="2.0.1"' '"jose"="4.15.5"' '"xlsx"="https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz"' '"tar"="6.2.1"' '"mysql2"="3.9.4"'; do
         jq ".resolutions.${override}" package.json > temp.json && mv temp.json package.json
```

Comment:
@hectorj2f @cpanato  sqlpad fails to start with failing to find mysql2. Is version 3.9.4 out and vendored inside the package?

Reply:
Maybe https://github.com/wolfi-dev/os/pull/19484 is better?